major-version-tag/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'major version tag'
+description: 'return the major version for a given full version'
+inputs:
+  version:
+    required: true
+outputs:
+  major_version:
+    description: 'major version of the provided full version'
+    value: ${{ steps.set-major-version.outputs.major_version }}
+runs:
+  using: 'composite'
+  steps:
+    - id: set-major-version
+      run: |
+        version=${{ inputs.version }}
+        MAJOR_VERSION=${version:0:2}.0
+        echo "major version is $MAJOR_VERSION"
+        echo "::set-output name=major_version::$MAJOR_VERSION"
+      shell: bash
```

Comment:
Please include newline at end of file.

(There's actually a standard for what-is-a-file, it wants this newline.)

Reply:
fixed